assignments/prototypes.js

```diff
@@ -102,9 +183,13 @@
   console.log(archer.greet()); // Lilith offers a greeting in Elvish.
   console.log(mage.takeDamage()); // Bruce took damage.
   console.log(swordsman.destroy()); // Sir Mustachio was removed from the game.
-*/
+  
+  console.log(javascriptWizard.challenge());
+  console.log(javascriptMonster.answer());
+  console.log(javascriptWizard.weapons[Math.floor(Math.random()*javascriptWizard.weapons.length)]);
 
```

Comment:
This is really impressive, Brittany! It's very cool that you chose to really challenge yourself with the stretch goal!